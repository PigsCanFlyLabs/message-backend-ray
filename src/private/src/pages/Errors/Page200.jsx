import React from 'react';

import LogoSVG from '@src/assets/logo.svg';

export default function Page200() {
  return (
    <div className="w-100 h-100 d-flex align-items-center justify-content-center flex-direction-column">
      <div className="d-flex align-items-center justify-content-center">
        <LogoSVG className="m-16" />
        <div className="sf-vertical-divider sf-vertical-divider-sm" />
        <h3 className="m-16 sf-text-color-primary">It Works!</h3>
      </div>
    </div>
  );
}
