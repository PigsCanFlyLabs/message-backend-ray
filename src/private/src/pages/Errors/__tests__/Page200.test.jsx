import React from 'react';

import Page200 from '../Page200';

describe('Page200', () => {
  it('should render without errors', () => {
    const component = <Page200 />;
    expect(component).toMatchSnapshot();
  });
});
