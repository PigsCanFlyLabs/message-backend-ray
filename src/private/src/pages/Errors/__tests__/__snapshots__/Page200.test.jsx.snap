// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page200 should render without errors 1`] = `<Page200 />`;
